rules/windows/defense_evasion_posh_obfuscation.toml

```
@@ -0,0 +1,104 @@
+[metadata]
+creation_date = "2024/07/03"
+integration = ["windows"]
+maturity = "production"
+updated_date = "2024/07/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies scripts that contain patterns and known methods that obfuscate PowerShell code. Attackers can use obfuscation
+techniques to bypass PowerShell security protections such as Antimalware Scan Interface (AMSI).
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-windows.powershell*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Potential PowerShell Obfuscated Script"
+reference = ["https://github.com/danielbohannon/Invoke-Obfuscation"]
```

```suggestion
references = ["https://github.com/danielbohannon/Invoke-Obfuscation"]
```

Should save you time trying to figure out the unit test error.